great_expectations_cloud/agent/actions/data_assistants/utils.py

```diff
@@ -91,9 +91,11 @@ def build_action_result(
     )
     context.add_expectation_suite(expectation_suite=expectation_suite)
 
-    checkpoint_config = {
-        "name": checkpoint_name,
-        "validations": [
+    checkpoint = context.add_checkpoint(
```

Comment:
Changed this to pass keyword arguments instead of dictionary unpacking to resolve these typing issues.

https://github.com/great-expectations/cloud/pull/344#issuecomment-2229333854